modules/homa-lite/src/benchmarking.rs

```diff
@@ -73,7 +73,9 @@ benchmarks! {
 
 	set_total_staking_currency {}: _(RawOrigin::Root, 1_000_000_000_000)
 
-	adjust_total_staking_currency {}: _(RawOrigin::Root, AmountOf::<T>::default())
+	adjust_total_staking_currency {}: _(RawOrigin::Root, AmountOf::<T>::max_value())
+
+	adjust_available_staking_balance {}: _(RawOrigin::Root, AmountOf::<T>::max_value())
```

Comment:
the weight depends on number of matched requests, which needs to be reflected on benchmarks

Reply:
Reworked how weights are calculated so `adjust_total_staking_currency` takes into account the number of matches this call "allows"